make-interactive-image

```diff
@@ -1,47 +1,29 @@
-#!/usr/bin/python
+#!/bin/bash
 
 # Tool to create interactive KVM images from our VMU images
 # Usage: make-interactive-image <VMU image filename> <interactive image filename>
 
-import guestfs
-import shutil
-import os
-import sys
-import tempfile
+INPUT_IMG=$1
+OUTPUT_IMG=$2
 
-if len(sys.argv) != 3:
-    print 'usage: %s <input image> <output image>' % sys.argv[0]
-    sys.exit(1)
+if [ $# -ne 2 ]; then
+    echo "Usage: make-interactive-image <VMU image filename> <interactive image filename>"
+    exit 1
+fi
 
-input_image = sys.argv[1]
-output_image = sys.argv[2]
+echo "Copying image..."
+cp $1 $2
```

Comment:
if we are setting `$INPUT_IMG` and `$OUTPUT_IMG`, we should probably use them instead of `$1` and `$2`, right?

Also please quote input parameters (`"$var"`) that get passed as arguments to other commands to prevent the shell interpreting whitespace / glob characters.